### DISCUSSION TITLE
Blazor Bootstrap in .NET MAUI Blazor Hybrid

### DISCUSSION THREAD

**Discussion by emnu1971** [vikramlearning/blazorbootstrap#349] [Q&A]
Hi,

Is it possible to use BlazorBootstrap from a .NET MAUI Blazor Hybrid app, and if so
which installation instructions must i follow (i guess similar as the webassembly setup as provided by the docs found here:
https://getblazorbootstrap.com/docs/getting-started/blazor-webassembly)

Thx for your feedback,

Kr,
Emmanuel.

**Comment by gvreddy04** [comment]
@emnu1971  Yes, follow the same instructions. Let me know, If you need any help.
